modules/nf-core/ampir/tests/main.nf.test

```diff
@@ -0,0 +1,65 @@
+nextflow_process {
+
+    name "Test Process AMPIR"
+    script "../main.nf"
+    process "AMPIR"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "ampir"
+
+    test("sarscov2 - bam") {
```

Comment:
It looks like the file used is actually candidatus_portiera_aleyrodidarum - proteome_fasta

Reply:
ah yes..thanks for spotting